backend/src/apiserver/main.go

```diff
@@ -149,6 +149,12 @@ func registerHttpHandlerFromEndpoint(handler RegisterHttpHandlerFromEndpoint, se
 	endpoint := "localhost" + *rpcPortFlag
 	opts := []grpc.DialOption{grpc.WithInsecure()}
 
+	// Only change the maxCallRecvMesSize if it is for visualizations
+	if serviceName == "Visualization" {
+		maxCallRecvMsgSize := 50 * 1024 * 1024
```

Comment:
Would you mind add a little bit more comments to this? E.g, default message size is 4MB and also a comment to indicate that the 50 * 1024 * 1024 is how many MB . Thanks!
Other than that /lgtm

Reply:
Done, sorry for late response